reddit_decider/__init__.py

```diff
@@ -19,8 +21,19 @@
 logger = logging.getLogger(__name__)
 
 EMPLOYEE_ROLES = ("employee", "contractor")
-EVENT_TYPE = "expose"
 
+class EventType(Enum):
+    EXPOSE = "expose"
+
+@dataclass
+class ExperimentConfig:
+    id: int
+    version: str
+    name: str
+    bucket_val: str
+    start_ts: int
+    stop_ts: int
+    owner: str
```

Comment:
The dataclass is necessary because the logger uses `getattr()` (e.g. `getattr(experiment, "name", None)` [here](https://github.snooguts.net/reddit/reddit-service-graphql/blob/3c5b239755b8ffb5770bfaa5ef5f5fd9e5e10635/graphql-py/graphql_api/events/utils.py#L218)).